src/main/features/response/draft/responseDraft.ts

```diff
@@ -125,4 +147,17 @@ export class ResponseDraft extends DraftDocument {
       (this.rejectPartOfClaim.option === RejectPartOfClaimOption.PAID_WHAT_BELIEVED_WAS_OWED ||
         this.rejectPartOfClaim.option === RejectPartOfClaimOption.AMOUNT_TOO_HIGH)
   }
+
+  private isResponseRejectedFullyWithAmountClaimed (): boolean {
```

Comment:
That does not read well. A `isResponseRejectedFullyWithAmountClaimedPaid` reads better.